code/modules/power/supermatter/supermatter.dm

```diff
@@ -682,7 +689,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		message_admins("[src] has consumed [key_name_admin(user)] [ADMIN_JMP(src)].")
 		investigate_log("has consumed [key_name(user)].", INVESTIGATE_ENGINES)
 		user.dust(force = TRUE)
-		if(power_changes)
+		if(is_power_processing())
 			matter_power += 200
```

Comment:
Events such as consumption should still increase the power, they are single events and aren't tied to processing

Reply:
But I can still leave the check on lasers, yes?

Reply:
Yea, since they are tied to a processing system (SSmachines)